Packs/DelineaSS/Integrations/DelineaSS/DelineaSS.py

```diff
@@ -476,7 +477,60 @@ def secret_rpc_changepassword_command(client, secret_id: str = '', newpassword:
     )
 
 
-def main():  # pragma: no cover
+def user_fetch_command(client, secretids):
+    credentials = fetch_credentials(client, secretids)
+    markdown = tableToMarkdown('Credentials to fetch', credentials)
+
+    return CommandResults(
+        readable_output=markdown,
+        outputs_prefix="Delinea.User.Fetch.Credentials",
+        outputs_key_field="credentials",
+        raw_response=credentials,
+        outputs=credentials
+    )
+
+
+def fetch_credentials(client, secretids):
+    credentials: List[Any] = []
+    finalsecretsid: List[Any] = []
+    finalcredentials: List[Any] = []
+    try:
+        secretsid = argToList(secretids)
+    except Exception as e:

```

Comment:
Please change exception to DemistoException instead.

Reply:
Resolved